includes/apple-exporter/class-theme.php

```diff
@@ -1552,12 +1563,16 @@ public function get_last_error() {
 	}
 
 	/**
-	 * Get the computed layout columns.
+	 * If user override enabled return layout_columns theme value.
+	 * Else return the computed layout columns value.
 	 *
 	 * @access public
 	 * @return int The number of layout columns to use.
 	 */
 	public function get_layout_columns() {
+		if ( 'yes' === $this->get_value( 'layout_columns_override' ) ) {
+			return $this->get_value( 'layout_columns' );
+		}
 		return ( 'center' === $this->get_value( 'body_orientation' ) ) ? 9 : 7;
 	}
```

Comment:
`layout_columns` was never a theme value.  It was calculated dynamically by `get_layout_columns()`.  Means I had to make a user configured value elective with an override.

Opting in returns the user configured `layout_columns`, opting out (the default setting) returns the computed value (original behavior).